static/gsApp/components/powerFeatureHovercard.tsx

```diff
@@ -104,9 +104,13 @@ class PowerFeatureHovercard extends Component<Props> {
                   ? t('Better With %s Plan', planName)
                   : t('Requires %s Plan', planName)}
               </div>
-              <LearnMoreLink onClick={this.handleClick} data-test-id="power-learn-more">
+              <ExternalLink
```

Comment:
Should it be LinkButton with priority link lol

Reply:
I've went ahead and changed this to use a button + the lightning icon
![CleanShot 2025-06-30 at 14 35 06@2x](https://github.com/user-attachments/assets/d56feca1-63dc-4568-af64-b1b747234e4f)